deployment/src/main/java/com/vaadin/quarkus/deployment/VaadinQuarkusProcessor.java

```diff
@@ -144,7 +145,11 @@ void mapVaadinServletPaths(final BeanArchiveIndexBuildItem beanArchiveIndex,
                     .builder(QuarkusVaadinServlet.class.getName(),
                             QuarkusVaadinServlet.class.getName())
                     .addMapping("/*").setAsyncSupported(true)
-                    .setLoadOnStartup(1).build());
+                    .setLoadOnStartup(1)
+                    // TODO: should be fixed in Flow and removed from here
+                    .addInitParam(ApplicationConfig.JSR356_PATH_MAPPING_LENGTH,
```

Comment:
Do still need this parameter? Flow's PushRequestHandler now sets it already.

Reply:
Good catch! I'll remove it

Reply:
done